src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts

```diff
@@ -61,6 +63,13 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
   public ngOnInit(): void {
     this.id = this.route.snapshot.params.id;
     this.error$ = this.hearingStore.select(fromHearingStore.getHearingActualsLastError);
+    this.hearingStore.select(fromHearingStore.getHearingValues).pipe(
+      filter((state: HearingValuesStateData) => !!state.serviceHearingValuesModel)
+    ).subscribe(
+      (state: HearingValuesStateData) => {
+        this.isPaperHearing = state.serviceHearingValuesModel.hearingChannels.indexOf('ONPPRS') > -1;
+      }
+    );
```

Comment:
it should grab the data from HearingRequestMainModel.hearingDetails.hearingChannels but not serviceHearingValuesModel. Please chat with @MateiHMCTS he is working on loading the value when the journey is started.